app/packages/core/src/components/ColorModal/ColorFooter.tsx

```diff
@@ -100,7 +104,7 @@ const ColorFooter: React.FC = () => {
                     );
 
                   const record = store.get(fragment.id);
-                  record && config?.setLinkedRecord(record, "colorScheme");
+                  record && config!.setLinkedRecord(record, "colorScheme");
```

Comment:
does this mean we want to ignore the fact that `config` can be undefined? maybe we should return on line 97

Reply:
`config` cannot be undefined. I will add a return.